docs/getting-started/building.md

```diff
@@ -0,0 +1,81 @@
+# Building Zarhus OS
+
+<!--
+TODO: scalable way to build different images (e.g. with WebKit or without
+WebKit support) for different platforms.
+-->
+
+This guide will demonstrate how to build a Zarhus OS image from zero!
+
+## Prerequisites
+
+* Linux PC (tested on `Ubuntu 20.04 LTS`)
```

Comment:
It was, long time ago actually. We do use Fedore in our team, but these documentation was added when we were using Ubuntu. I have not changed because it will need some verification then.

Reply:
meta-zarhus first commit was 3 months ago I'm asking because I'm sure those docs were taken from very old template and it wasn't tested with meta-zarhus. I'm pretty sure most of us built it at least once and it was always on Fedora so it is verified. Ubuntu on the other hand isn't

Reply:
It builds inside `kas-container`, so there should not be any problems with building it on Ubuntu.

Reply:
I know and I'm not asking only about that but everything else, will kas work, will bmaptools work, will any of the other steps work? In another document - `radxa-cm3` it's also implied that Ubuntu is prerequisite (use of `apt-get`).
Did you do those steps on Ubuntu (20.04) or Fedora?
It also makes maintenance harder if none of meta-zarhus developers use Ubuntu.
Can you ping someone who built this on Ubuntu so they can confirm it works?
If not then I can't really agree to `tested on ...` if it wasn't tested on this platform.

> but this documentation was added when we were using Ubuntu
I'm not sure I follow, you are adding this documentation just now.

Reply:
> Can you ping someone who built this on Ubuntu so they can confirm it works?

Good argument, heh. Ok, I will change it to Fedora and verify.

Reply:
Migrated here 0263e6325e56da2ad60f5c9fa769f8dcec8357d9.